src/WebJobs.Script/Description/CSharp/FunctionEntryPointAttribute.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    [System.AttributeUsage(AttributeTargets.Method, Inherited = false, AllowMultiple = false)]
+    public sealed class FunctionEntryPointAttribute : Attribute
```

Comment:
Why would there be multiple public methods? What is the scenario for that? Helper methods would be private, right? Wondering if we can't just keep this simple/convention based. If there is a single public method, that is the entry point. If there are multiple public methods, the one called Run is the entry point. Not sure I see the need for the attribute.